### PR TITLE
feat: #3523 scaffolder update

### PR DESCRIPTION
**Pull request by willmcvay**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #3523

### CODE REVIEW

File: packages/react-app-scaffolder/app/templates/_config.json [hunk 1]
```diff
@@ -2,10 +2,9 @@
...
-  "connectUserPoolId": "",
+  "connectUserPoolId": "<%= userPoolId %>",
   "googleAnalyticsKey": "",
-  "connectOAuthUrl": "",
-  "platformApiUrl": "",
-  "graphqlUri": "",
-  "marketplaceUrl": ""
+  "connectOAuthUrl": "https://connect.dev.paas.reapit.cloud",
+  "platformApiUrl": "https://platform.dev.paas.reapit.cloud",
+  "marketplaceUrl": "https://marketplace.dev.paas.reapit.cloud"
```
Comment (bashleigh) [inline packages/react-app-scaffolder/app/templates/_config.json:9]: Are there instructions on how to go live for this? Just double making sure

File: packages/react-app-scaffolder/app/templates/react-app/src/components/pages/login-page.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  <LoginContainer>
+    <LoginImageContainer>
+      <LoginImage src={loginGraphic} alt="login graphic" />
+    </LoginImageContainer>
+    <LoginWrapper>
+      <BodyText>
+        <img src={connectImage} alt="Reapit Connect Graphic" />
+      </BodyText>
+      <BodyText>
+        <Button type="button" onClick={loginHandler} loading={false} intent="primary" disabled={false} fullWidth>
```
Comment (bashleigh) [inline packages/react-app-scaffolder/app/templates/react-app/src/components/pages/login-page.tsx:20]: disabled and loading are false by default?

File: packages/react-app-scaffolder/app/templates/_config.json (hunk 1 above)
Comment (bashleigh) [reply packages/react-app-scaffolder/app/templates/_config.json:9]: Or is this for mono only?

Comment (willmcvay) [reply packages/react-app-scaffolder/app/templates/_config.json:9]: It's Mono Only

File: packages/react-app-scaffolder/app/templates/react-app/src/components/pages/login-page.tsx (hunk 2 above)
Comment (willmcvay) [reply packages/react-app-scaffolder/app/templates/react-app/src/components/pages/login-page.tsx:20]: Fair point

**Pull request merged**